### ISSUE TITLE
Link login modal to 'subscribe button' on flash message

### ISSUE THREAD

**Issue by oorjitchowdhary** [publiclab/plots2#4433]
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

![50456435-75e34700-097a-11e9-9905-e635d371a465](https://user-images.githubusercontent.com/39333058/50515438-cd90c800-0aca-11e9-9a7f-1640f04e4f12.png)

- [ ] 📝 **Update** the file `app/views/questions/show.html.erb` in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

https://github.com/publiclab/plots2/blob/f778b54f7a9b32569b92f501f4f60df695843d37/app/views/questions/show.html.erb#L23-L24

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
-  <a class="btn btn-default" target="_blank" href="/subscribe/tag/question:<%= pt %>">
+ <a class="btn btn-default requireLogin" target="_blank" href="/subscribe/tag/question:<%= pt %>">
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

**Comment by arungoel123456** [comment]
@oorjitchowdhary  Can I work on this issue

**Comment by SidharthBansal** [comment]
@arungoel123456 please go ahead. @oorjitchowdhary is on holidays till 9th January so he was not able to respond. I am really sorry to wait you so long.

**Comment by SidharthBansal** [comment]
@arungoel123456 this issue is solved you can try  #4364 if you want.
